### PR TITLE
Package build changes 

### PR DESCRIPTION
**Pull request by jquense**
This moves the build command to each package, this should simplify a few test issues as well as make publishing a bit less error prone.

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit 6fcc2df37d652e3e42b91929bb715b61ca750789

https://app.netlify.com/sites/using-glamor/deploys/59a80a7c424ef2045f38735b

**Comment by jquense** [comment]
This should solve some CI issues also

File: packages/gatsby-source-filesystem/src/__tests__/create-file-node.js [hunk 1]
```diff
@@ -2,20 +2,23 @@ const path = require(`path`)
 
 const { createFileNode } = require(`../create-file-node`)
 
-describe(`create-file-node`, () => {
-  it(`creates a file node`, done => {
-    createFileNode(
-      path.resolve(`${__dirname}/fixtures/file.json`),
-      {},
-      (err, fileNode) => {
```
Comment (KyleAMathews) [inline packages/gatsby-source-filesystem/src/__tests__/create-file-node.js]: Oh right :-)

Comment (jquense) [reply packages/gatsby-source-filesystem/src/__tests__/create-file-node.js]: we need like a schema based testing thing for this i think. like `matchesSortOfThis()` :P

Comment (KyleAMathews) [reply packages/gatsby-source-filesystem/src/__tests__/create-file-node.js]: `matchesSnapshotWithinLevenshteinDistanceOf(10)`

**Pull request merged**